crates/agglayer-node/src/rpc/tests.rs

```diff
@@ -106,6 +107,52 @@ async fn check_tx_status() {
     assert_eq!(res, "done");
 }
 
+#[tokio::test]
+async fn check_tx_status_fail() {
```

Comment:
We could add some tests with `insta` to acknowledge the shape of the errors.

Reply:
Added a bunch of them covering all the error cases defined in the `rpc::error` module.